examples/kubernetes-features/README.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Azure Container Service Engine - Kubernetes Features
```

Comment:
But maxPods is not a networking setting. It is only indirectly related because it affects number of IP addresses.

@anhowe @dmitsh Can you please advise on how this document tree should be structured?